linode/manager
# M3-4279: Remove duplicate text truncation helpers

## Description

Combine both text truncation utilities into one file. The original intent was to get rid of `truncateText.ts` but I decided to merge the two instead since it breaks the text at the end of the word whereas the functions in `truncate.ts` breaks the text mid-word

## How to test

Check everywhere that `truncateText` is be called to make sure there's no regression
